tests/functional/firefox/whatsnew/test_whatsnew_73.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.common.exceptions import TimeoutException
+from pages.firefox.whatsnew.whatsnew_73 import FirefoxWhatsNew73Page
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_get_mobile_modal_displayed(base_url, selenium):
+    page = FirefoxWhatsNew73Page(selenium, base_url, locale='sv-SE').open()
```

Comment:
This test will fail in production unless the `sv-SE` locale is activated, just a heads up

Reply:
Would there be a better way to write this test? Unsure how to write a condition for "has firefox set as default already" specifically...

Reply:
Keep in mind we can't do automated tests for things that rely on UItour, or browser features that depend on system settings. If this CTA is shown to Firefox users only if it's default then that may cause issues in automation.

Also, please make sure to push your tests to run-integration-tests branch and make sure they pass.

Reply:
All of these tests are conditional on uitour, I'm thinking I should remove the tests altogether?

Reply:
You can test whatever the default state for the page is.